composer.json

```diff
@@ -22,21 +24,16 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": "^8.2",
-        "phpstan/phpstan": "^1.10.55",
-        "illuminate/support": "^10|^11",
-        "laravel/pint": "^1.13.9"
```

Comment:
the PHPStan and Laravel Pint packages are used during development. So made more sense to have them in the `require-dev`

Reply:
`"illuminate/support": "^10|^11",` would need to stay, as without it you'd be able to install the package in Laravel < 10 (or even, not in a Laravel app)